solo8:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: "position_controllers/JointPositionController"
    joint: FL_HFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint2_position_controller:
    type: "position_controllers/JointPositionController"
    joint: FL_KFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint3_position_controller:
    type: "position_controllers/JointPositionController"
    joint: FR_HFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint4_position_controller:
    type: "position_controllers/JointPositionController"
    joint: FR_KFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint5_position_controller:
    type: "position_controllers/JointPositionController"
    joint: HL_HFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint6_position_controller:
    type: "position_controllers/JointPositionController"
    joint: HL_KFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint7_position_controller:
    type: "position_controllers/JointPositionController"
    joint: HR_HFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  joint8_position_controller:
    type: "position_controllers/JointPositionController"
    joint: HR_KFE
    pid: {p: 100.0, i: 0.01, d: 10.0}

  #Para habilitar o controle PID (Atualmennte muito instavel)
  #gazebo_ros_control:
  # pid_gains:
  #    FL_HFE: {p: 30, i: 0.0, d: 0.01}
  #    FL_KFE: {p: 30, i: 0.0, d: 0.01}
  #    FR_HFE: {p: 30, i: 0.0, d: 0.01}
  #    FR_KFE: {p: 30, i: 0.0, d: 0.01}
  #    HL_HFE: {p: 30, i: 0.0, d: 0.01}
  #    HL_KFE: {p: 30, i: 0.0, d: 0.01}
  #    HR_HFE: {p: 30, i: 0.0, d: 0.01}
  #    HR_KFE: {p: 30, i: 0.0, d: 0.01}